### PR TITLE
Add parse@1.9.2 w/ npm auto-update

### LINKED ISSUE #9738: [Request] Add parse
**Library name:** parse
**Git repository url:** https://github.com/ParsePlatform/Parse-SDK-JS
**npm package url(optional):**  https://www.npmjs.com/package/parse
**License(s):**
**Official homepage:** https://github.com/ParsePlatform/Parse-SDK-JS
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #9738

Some old versions(v1.6.0, v.1.6.1, v1.6.3) of parse can not be got because they were not packaged by the authors.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
 * Git repository (required): https://github.com/ParsePlatform/Parse-SDK-JS
 * Official website (optional, not the repository): https://www.parse.com
 * NPM package url (optional): https://www.npmjs.com/package/parse
 * License and its reference: BSD-3-Clause
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 506
   - Count of watchers: 60
   - Count of forks: 290
 * NPM download stats (optional):
   - Downloads in the last day: 2588
   - Downloads in the last week: 15527
   - Downloads in the last month: 89380

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9738, cc @ParsePlatform

### CODE REVIEW

**Comment by extend1994** [comment]
@PeterDaveHello please help me review this pull request, thank you.

**Comment by extend1994** [comment]
@PeterDaveHello please help me review this pull request, thank you.

File: ajax/libs/parse/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "type": "git",
+    "url": "https://github.com/ParsePlatform/Parse-SDK-JS"
+  },
+  "npmName": "parse",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "parse*"
+      ]
```
Comment (PeterDaveHello) [inline ajax/libs/parse/package.json]: Isn't this auto-update config old versions only?

**Comment by extend1994** [comment]
@PeterDaveHello I modified `package.json`, please help me review it again, thank you.

**Pull request merged**